.github/workflows/lint.yaml

```diff
@@ -55,7 +55,11 @@ jobs:
 
       - name: get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "dir=$(pwd)/.yarn/cache" >> $GITHUB_OUTPUT
+        run: |
```

Comment:
I don't get the changes though.

Reply:
This was a testing to check that the new commands where available in the runner. It's been reverted